example/sound.cpp

```diff
@@ -0,0 +1,80 @@
+#include <al.h>
+#include <alc.h>
+#define DR_WAV_IMPLEMENTATION
+#include <dr_libs/dr_wav.h>
+
+#include <iostream>
+#include <vector>
+
+int main(int argc, char** argv) {
+	if (argc != 2) {
+		std::cerr << "Syntax: " << argv[0] << " <wav file path>" << std::endl;
+		return -1;
+	}
+
+	std::vector<int16_t> sample_data;
+	ALsizei sample_rate;
+	ALenum sample_format;
+	{
+		drwav wav;
+		if (!drwav_init_file(&wav, argv[1], NULL)) {
+			std::cerr << "Invalid file inputted" << std::endl;
+			return -1;
+		}
+
+		sample_rate = static_cast<ALsizei>(wav.sampleRate);
+		sample_data.resize((size_t)wav.totalPCMFrameCount * wav.channels * sizeof(int16_t));
```

Comment:
`sizeof(int16_t)` doesn't seem correct, `resize` should take the number of elements, not bytes.

Reply:
oops, left that from the previous malloc